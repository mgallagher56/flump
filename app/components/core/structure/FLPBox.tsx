import React from 'react';

import type { BoxProps } from '@chakra-ui/react';
import { Box } from '@chakra-ui/react';

const FLPBox: React.FC<React.PropsWithChildren<BoxProps>> = (props) => {
  return <Box {...props}>{props.children}</Box>;
};

export default FLPBox;
