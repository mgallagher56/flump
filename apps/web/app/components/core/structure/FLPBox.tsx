import type { FC, PropsWithChildren } from 'react';

import type { BoxProps } from '@chakra-ui/react';
import { Box } from '@chakra-ui/react';

const FLPBox: FC<PropsWithChildren<BoxProps>> = (props) => {
  return <Box {...props}>{props.children}</Box>;
};

export default FLPBox;
