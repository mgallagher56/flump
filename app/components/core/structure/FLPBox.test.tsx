import { render } from '@testing-library/react';
import { describe, expect, test } from 'vitest';

import FLPBox from './FLPBox';

describe('FLPBox', () => {
  test('renders', () => {
    const { container, getByText } = render(
      <FLPBox>
        <div>FLPBox</div>
      </FLPBox>
    );
    const linkElement = getByText('FLPBox');
    expect(linkElement).toBeDefined();
    expect(container).toMatchSnapshot();
  });
});
