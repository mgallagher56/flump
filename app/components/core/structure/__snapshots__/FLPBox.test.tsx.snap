// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`FLPBox > renders 1`] = `
<div>
  <div
    class="css-0"
  >
    <div>
      FLPBox
    </div>
  </div>
</div>
`;
